### PR TITLE
Plaza exec endpoint added

### PR DESCRIPTION
**Pull request by willrnch**
This add the `/exec` endpoint to plaza that executes shell commands as the specified user.

### CODE REVIEW

File: plaza/routes/exec/exec.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	}
+
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err = cmd.Run()
+
+	// if err != nil {
```
Comment (dynamiccast) [inline plaza/routes/exec/exec.go:50]: Why is this code commented ?

File: plaza/windows/syscall.go [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+	environment uintptr,
+	currentDirectory string,
+	si *syscall.StartupInfo,
+	pi *syscall.ProcessInformation,
+) error {
+	fmt.Println("createProcessWithLogon")
+	fmt.Println(cmd)
+	dll, err := syscall.LoadDLL("advapi32.dll")
+	if err != nil {
+		fmt.Println("lol")
```
Comment (dynamiccast) [inline plaza/windows/syscall.go]: Please remove debug prints

File: plaza/windows/process.go [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+	if e != nil {
+		return nil, os.NewSyscallError("GetExitCodeProcess", e)
+	}
+	var u syscall.Rusage
+	e = syscall.GetProcessTimes(syscall.Handle(handle), &u.CreationTime, &u.ExitTime, &u.KernelTime, &u.UserTime)
+	if e != nil {
+		return nil, os.NewSyscallError("GetProcessTimes", e)
+	}
+	p.setDone()
+	// NOTE(brainman): It seems that sometimes process is not dead
```
Comment (dynamiccast) [inline plaza/windows/process.go:113]: If this code was imported from an other open-source project, then @Gentux should add it to the list of third party library.

File: plaza/routes/exec/exec.go (hunk 1 above)
Comment (willrnch) [inline plaza/routes/exec/exec.go:50]: When the process fails (Exit code != 0), it returns an error but usefull informations are in stdout (for powershell for instance). But the error can be something else that an invalid exit code. We should deal with that case in the future.

File: plaza/windows/process.go (hunk 3 above)
Comment (willrnch) [inline plaza/windows/process.go:113]: Imported from golang std lib `https://github.com/golang/go/blob/0da4dbe2322eb3b6224df35ce3e9fc83f104762b/src/os/exec_windows.go#L38`

Comment (dynamiccast) [inline plaza/windows/process.go:113]: Got it @Gentux ?

**Pull request merged**